ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/default_interfaces.schema.yml

```diff
@@ -18,7 +18,10 @@ keys:
         types:
           type: list
           required: true
-          description: List of node type keys.
+          description: |
+            List of node type keys.
+            'default' will apply these default interfaces to node types where there is no more-specific match.
+            Note that the node's platform must still match a platform type regular expression.
```

Comment:
which can be `default` right (if not overwritten)?

Reply:
Yup, will amend docs to make that clear.